website/source/docs/connect/proxies/envoy.md

```diff
@@ -359,6 +378,13 @@ The following configuration items may be overridden directly in the
 definition](/docs/connect/registration/service-registration.html) or
 [`sidecar_service`](/docs/connect/registration/sidecar-service.html) block.
 
+~> **Note:** - When a
+[`service-router`](/docs/agent/config-entries/service-router.html),
+[`service-splitter`](/docs/agent/config-entries/service-splitter.html), or
+[`service-resolver`](/docs/agent/config-entries/service-resolver.html) config
+entry exists for a service these escape hatches are ignored and will log a
```

Comment:
```suggestion
entry exists for a service the below escape hatches are ignored and will log a
```

Reply:
This explanation looks good, but we might want to also mention that this will happen in the discovery chain documentation in case someone already relies on one of these hatches and wants to use the discovery chain too.

Reply:
I mentioned it in the same place that we mention incompatibilities with built-in and native proxies.